### DISCUSSION TITLE
Trouble doing the export step.

### DISCUSSION THREAD

**Discussion by rafikiphoto** [vzhd1701/evernote-backup#61] [General]
I have completed the sync step but I am having trouble doing the export step. 
My evernote-backup.exe and en_backup.db are in C:\Users\brian\downloads\bin_evernote_backup. 
I am attempting to export to my Proton Drive folder: C:\Users\brian\Proton Drive\brian.davies\My files\Github Evernote Backup/
To export I enter in Powershell: 
C:\Users\brian\downloads\bin_evernote_backup> .\evernote-backup export 'C:\Users\brian\Proton Drive\brian.davies\My files\Github Evernote Backup/' and I get an error:  
Missing argument 'OUTPUT_PATH'.
Could someone guide me in formulating the correct command, please?

**Comment by vzhd1701** [comment]
Sorry, but I cannot reproduce your error. Your command looks file, maybe it has something to do with Windows Defender and user files protection. Could you try exporting into different directory?

**Comment by casey-riley** [comment]
Yeah Powershell's ALWAYS annoying & finnicky to deal with. But one way or another, it's a windows issue, and not a problem with the program;

Bc it's saying the error is the ouput path variable, so for whatever reason it doesn't know/understand where you want to save the output.

Anyway -

- [ ]  First, did this work? 

> Could you try exporting into different directory?

If not, let's try this...
---
Here's your original command again for reference, so I can point out the little tweaks I made. **(changes look like this)**

> C:\Users\brian\downloads\bin_evernote_backup> .\evernote-backup export 'C:\Users\brian\Proton Drive\brian.davies\My files\Github Evernote Backup/'

Reasonably, this "should" work, keyword being should... 

All I did, was switch out the quotes to be slightly more explicit about the path having spaces, and where those are. 



- [ ]  This one has doubles to outline the path, and singles for where the first directory with a space is in the path, and one for after the last space 

`C:\Users\brian\downloads\bin_evernote_backup> .\evernote-backup export "C:\Users\brian\'Proton Drive\brian.davies\My files\Github Evernote Backup'/"`


- [ ]  and this one has double quotes to outline the path, and singles for each directory with a space


`C:\Users\brian\downloads\bin_evernote_backup> .\evernote-backup export "C:\Users\brian\'Proton Drive'\brian.davies\'My files'\'Github Evernote Backup'/"
`

---
---
Hopefully one of these works out for you, but if not, here's an article on dealing with spaces in the path in powershell, you might want to look into each of these options

https://www.delftstack.com/howto/powershell/powershell-path-with-spaces/

If none of that works, You can also try (temporarily) renaming "Proton Drive", to Proton-Drive or Proton_Drive, or something like that... and do the same for any other names with spaces. 

After all of that... at the very least, we can completely rule out spaces as the issue in the path variable 😂😂
